source/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -77,10 +77,27 @@ By default Cypress will automatically include the plugins file `cypress/plugins/
 
 ## Support file
 
-By default Cypress will automatically include the support file `cypress/support/index.js` **before** every single spec file it runs. We do this purely as a convenience mechanism so you don't have to import this file in every single one of your spec files.
+By default Cypress will automatically include the support file `cypress/support/index.js`. This file runs **before** every single spec file . We do this purely as a convenience mechanism so you don't have to import this file in every single one of your spec files.
```

Comment:
typo possibly?

Reply:
Yes. Perfect, that previous wording is definitely confusing